tests/test_ballot_generator.py

```diff
@@ -48,7 +49,20 @@ def test_PL_completion():
     )
     profile = pl.generate_profile(number_of_ballots=100)
     assert type(profile) is PreferenceProfile
-
+    
+def test_SP_completion():
+    pl = SlatePreference(
+        candidates=["W1", "W2", "C1", "C2"],
+        slate_to_candidates={"W": ["W1", "W2"], "C": ["C1", "C2"]},
+        pref_interval_by_bloc={
+            "W": {"W1": 0.4, "W2": 0.3, "C1": 0.2, "C2": 0.1},
+            "C": {"W1": 0.2, "W2": 0.2, "C1": 0.3, "C2": 0.3},
+        },
+        bloc_voter_prop={"W": 0.7, "C": 0.3},
+        cohesion_parameters={"W": 0.7, "C": 0.9},
+    )
+    profile = pl.generate_profile(number_of_ballots=100)
+    assert type(profile) is PreferenceProfile
```

Comment:
should test verify that the PreferenceProfile is the expected one here?

Reply:
Good call